app/forms/nfg_onboarder/informational_form.rb

```diff
@@ -1,5 +1,3 @@
 class NfgOnboarder::InformationalForm < Reform::Form
   model NfgOnboarder::InformationalForm
-
-  def build_errors; end
```

Comment:
The initial intention of adding this method was to allow us to run an onboarder in the spec/dummy app. with the recent version of reform we no longer required this hack.  
This hack causes issues on form submission, it will return nil for self.errors attribute. 
<img width="1602" alt="Screen Shot 2020-09-24 at 11 31 53 AM" src="https://user-images.githubusercontent.com/717069/94106734-a6ab2580-fe59-11ea-9eca-8a806d3e08d2.png">